pex/resolve/locker_patches.py

```diff
@@ -66,6 +68,8 @@ def _get_env(environment, name):
             return python_versions_strings
         if name == "python_full_version":
             return python_full_versions_strings
+        if name == "os_name":
```

Comment:
For anyone willing to check my work, the official list of possible markers: https://peps.python.org/pep-0508/#environment-markers

I'm pretty sure I have all the relevant ones to `--target-system` covered.

Reply:
Checked your work and I believe you are correct.

Reply:
Excellent. Thank you.